pkg/policy/api/rule_validation.go

```diff
@@ -640,16 +640,12 @@ func (pr *PortDenyRule) sanitize() error {
 }
 
 func (pp *PortProtocol) sanitize(hasDNSRules bool) (isZero bool, err error) {
-	if pp.Port == "" {
```

Comment:
Should we keep this restriction here if `!option.Config.EnableBPFConntrackUnknownProtocols`? Or are VRRP / IGMP always allowed?